### PR TITLE
perf: preload the geometry

### PR DESCRIPTION
**Pull request by paulirish**
Current loading waterfall:
image

The network request for the geometry data only starts once the plan json is done (red annotation). The pink annotation shows the 300ms-ish of opportunity.

If we start a preload for the geometry at the same time as we start the plan json, we save some time.
image

### CODE REVIEW

File: planscore/website/templates/plan.html [hunk 1]
```diff
@@ -180,6 +180,15 @@ <h4>Sensitivity Testing</h4>
 	        mm_metric_url = 'https://planscore.org/metrics/meanmedian/',
 	        model_url_pattern = '{{ url_for("get_model_description", prefix="data/2020") }}';
 
+        // preload the geometry as it can take a while to load
+        const probably_geometry_url = plan_url.replace('index.json', 'geometry.json');
```
Comment (paulirish) [inline planscore/website/templates/plan.html:184]: @migurski i'll want your feedback on this bit. :) 

in the cases i looked at, the url construction was this easy. But I see the `geom_prefix` there, so I suspect there's a more robust way of making this URL?

**Comment by migurski** [comment]
Nice! We can probably tell the front-end how to build that geometry URL based on this bit of the backend: https://github.com/PlanScore/PlanScore/blob/main/planscore/data.py#L7

Maybe pass a URL pattern into the page template and use it to build a search/replace?

**Comment by paulirish** [comment]
Thanks! The pointer was useful. Switched up the approach and seems better now.

**Comment by migurski** [comment]
This won’t work in production because we freeze everything down to static files for deployment, so Flask isn’t actually involved in a request path! Sorry if that’s confusing, I can pick up this PR if you’d like?

**Comment by paulirish** [comment]
> This won’t work in production because we freeze everything down to static files for deployment, so Flask isn’t actually involved in a request path! Sorry if that’s confusing, I can pick up this PR if you’d like?

ah gotcha. 

i gave it one more shot. ;)  if this doesnt do the trick, please feel free to take over the PR.

**Comment by migurski** [comment]
Works!

**Pull request merged**